app/containers/Applications/ApplicationCommentsByForm.tsx

```diff
@@ -35,25 +40,92 @@ export const ApplicationCommentsByForm: React.FunctionComponent<Props> = props =
     console.log(response);
   };
 
+  const deleteComment = async () => {
+    const {environment} = props.relay;
+    const variables = {
+      input: {
+        id: reviewComment.id,
+        reviewCommentPatch: {
+          deletedAt: moment
+            .tz('America/Vancouver')
+            .format('YYYY-MM-DDTHH:mm:ss'),
+          commentType: reviewComment.commentType
+        }
+      },
+      applicationId: reviewComment.applicationByApplicationId.id,
+      version: String(version)
+    };
+
+    const response = await deleteReviewCommentMutation(
+      environment,
+      variables,
+      formResultId
+    );
+    console.log(response);
+  };
+
   return (
     <>
       <tr>
-        <td>{reviewComment.description}</td>
+        <td>
+          <div className="description">{reviewComment.description}</div>
+          <div>
+            <small style={{color: '#777'}}>
+              {reviewComment.ciipUserByCreatedBy.firstName}{' '}
+              {reviewComment.ciipUserByCreatedBy.firstName} (
```

Comment:
should this be 
```suggestion
              {reviewComment.ciipUserByCreatedBy.lastName} (
```
?

Reply:
Ohh good catch, fixed